chainerrl/envs/ale.py

```diff
@@ -116,10 +118,9 @@ def current_screen(self):
         rgb_img = np.maximum(self.ale.getScreenRGB(), self.last_raw_screen)
         # Make sure the last raw screen is used only once
         self.last_raw_screen = None
-        assert rgb_img.shape == (210, 160, 3)
```

Comment:
Screen sizes can be different from (210, 160) when you use wrong rom files, so it's better to check them.

Reply:
This change removes a double-check.  This line checks rgb-format and L125--127 checks screen sizes.

Reply:
I see.